src/utils/helpers/structure/generateReviewSectionActions.ts

```diff
@@ -83,9 +100,14 @@ const generateReviewSectionActions: GenerateSectionActions = ({
   currentUserId,
 }) => {
   const isCurrentUserReview = reviewAssignment.reviewer.id === currentUserId
+  const isConsolidation = reviewAssignment.level > 1
 
   sections.forEach((section) => {
-    const isReviewable = (section.reviewAndConsolidationProgress?.totalReviewable || 0) > 0
+    const { totalReviewable, totalPendingReview, totalActive } = isConsolidation
```

Comment:
I was wondering what was the logic behind changing to combined progress (i.e. extra added code/logic and + testing + extra time taken to review refactor). not a required changed but I am trying to figure out what was the reason behind the change, likewise for the reviewAction -> assignment change

Reply:
Sorry if this has been difficult to review. The reason was only for readability, it seemed easier to see things as separated entities - either Reviewer or Consolidator. I know this was just changed by you for break-down the logic, but it wasn't helpful to me, because then I had to get things from 2 different places to come up with a progress.

Changes on  `reviewAction -> assignment` also for consistency. I needed to create a type - because I was using a variable that would be of this same type (not defined yet), then there was already a `ReviewAction` type, which was one of the fields of this old field on `SectionState`, so it used to be:
```
SectionState {
...
reviewAction: {
   action: ReviewAction
   isAssignedToCurrentUser
   etc
}
```
Because I wanted to use a type for this instance I didn't know how to name it properly, so changed it to assignment, maybe it would be more correct to be `sectionAssignment`. I agree it adds to this change, you are right. But it would be really difficult if every change wasn't allowed during development of each new feature when it's considered refactoring. Even if it's a refactoring of a variable name.